sdks/python/gen_protos.py

```diff
@@ -174,22 +239,20 @@ def write_message(self, message_name, message, indent=0):
         ctx.prepend('')
       return ctx.lines
 
-  pb2_files = [path for path in glob.glob(os.path.join(out_dir, '*_pb2.py'))]
-  api_path = os.path.dirname(pb2_files[0])
-  sys.path.insert(0, os.path.dirname(api_path))
+  pb2_files = list(glob.glob(os.path.join(out_dir, '*_pb2.py')))
 
-  def _import(m):
-    return import_module('api.%s' % m)
-
-  try:
-    beam_runner_api_pb2 = _import('beam_runner_api_pb2')
-    metrics_pb2 = _import('metrics_pb2')
+  with PythonPath(os.path.dirname(api_path), front=True) as _:
```

Comment:
The `as _` can be omitted (here and below).

Reply:
fixed